src/View/clientForm/clientFormView.java

```diff
@@ -0,0 +1,274 @@
+package View.clientForm;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import javax.swing.*;
+
+public class clientFormView extends JPanel implements ActionListener {
+    private JPanel panelFormMain;
+    private JLabel clientDetails;
+    private JLabel clientName;
+    private JLabel clientEmail;
+    private JLabel clientPhone;
+    private JLabel clientContact;
+    private JLabel clientAddress;
+    private JLabel clientCity;
+    private JLabel clientState;
+    private JLabel clientZIP;
+    private JLabel clientType;
+    private JTextField nameText;
+    private JTextField emailText;
+    private JTextField phoneText;
+    private JTextField contactText;
+    private JTextField addressText;
+    private JTextField cityText;
+    private JTextField stateText;
+    private JTextField zipText;
+    private JTextField typeText;
+    private JButton clientButton;
+
+    public clientFormView(String action) {
+
+        clientDetails = new JLabel("    Client Details");
+        clientName = new JLabel("Client Name: ");
+        clientEmail = new JLabel("Email : ");
+        clientPhone = new JLabel("Phone Number: ");
+        clientContact = new JLabel("Preferred Contact: ");
+        clientAddress = new JLabel("Address: ");
+        clientCity = new JLabel("City: ");
+        clientState = new JLabel("State: ");
+        clientZIP = new JLabel("ZIP: ");
+        clientType = new JLabel("Type: ");
+
+        nameText = new JTextField(20);
+        emailText = new JTextField(20);
+        phoneText = new JTextField(20);
+        contactText = new JTextField(20);
+        addressText = new JTextField(20);
+        cityText = new JTextField(12);
+        stateText = new JTextField(4);
+        zipText = new JTextField(10);
+        typeText = new JTextField(15);
+
+        if(action.equals("Read"))
+        {
+            nameText.setText("name");
+            emailText.setText("1234@email.com");
+            phoneText.setText("111-111-1111");
+            contactText.setText("Phone");
+            addressText.setText("1234 St.");
+            cityText.setText("Baltimore");
+            stateText.setText("MD");
+            zipText.setText("11111");
+            typeText.setText("Individual");
```

Comment:
setText looks good, when we connect to the Model we will change this. If you could add a comment on ln 55 so we know to fix this when we are ready to connect to the model